README.md

```diff
@@ -15,6 +15,16 @@ How to run this Ansible playbook
 pip install -r requirements.txt
 ```
 
+3. Launch the ansible playbook
+
+```bash
+ansible-playbook playbook.yml
```

Comment:
I'd give the playbook a meaningful name

Reply:
what are meaninful names for playbooks? :) I usually call it playbook.yml

Reply:
Do you call classes in Python just `Class`?

Reply:
If I have only one in my whole project, most likely yes.

Reply:
Don't. Also, you have different flows here. You have platform provisioning/bootstrapping and launching apps.